pack/valk.json

```diff
@@ -388,6 +420,7 @@
 		"position": 33,
 		"quantity": 3,
 		"resource_physical": 1,
+		"text": "Alliance.\n<b>Hero Action<b> (thwart): Exhaust an <b>AVENGER<b> character and a <b>GUARDIAN<b> character -> remove X threat from each scheme, where X is equal to the combined THW of those characters.",
```

Comment:
closing tags need to be </b> seem to have use <b> on both sides

Reply:
if thid comment was about blodness tags, I fixed the tags (sorry not a native english speaker).

Reply:
well whatever I wrote didnt make much sense, I did mean to say closing tags need to be </b>